components/Notes/index.js

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+
+export default function Notes({ recipeId }) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [note, setNote] = useState("");
+
+  useEffect(() => {
+    const savedNote = localStorage.getItem(`note-${recipeId}`);
+    if (savedNote) {
+      setNote(savedNote);
+      setIsOpen(true);
+    }
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem(`note-${recipeId}`, note);
+  }, [note]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+  };
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const handleChange = (event) => {
+    setNote(event.target.value);
+  };
+
+  return (
+    <>
+      <StyledButton type="button" onClick={handleToggle}>
+        {isOpen ? "Close Note" : "Add Note"}
+      </StyledButton>
+
+      {isOpen && (
+        <StyledForm onSubmit={handleSubmit}>
+          <StyledTextarea
+            value={note}
+            name="notes"
+            onChange={handleChange}
+            maxLength={500}
+          />
+          <br />
+          <StyledButton type="submit">Save</StyledButton>
+        </StyledForm>
+      )}
+    </>
+  );
+}
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const StyledButton = styled.button`
+  color: #f4f5f6;
+  background-color: #0f5c64;
+  border: 1px ridge #bdc0bf;
+  border-radius: 45px;
+`;
+
+const StyledTextarea = styled.textarea`
+  border: 1px ridge #bdc0bf;
+  height: ${({ isOpen }) => (isOpen ? "200px" : "50px")};
+  width: 100%;
+  resize: none;
+  font-family: "Nunito", sans-serif;
+`;
+
+//klappt noch nicht, dass jedes rezept verschiedene notes hat
+
+//braucht noch exit icon
```

Comment:
Is this part of the current user story? If not, you can directly write a new one :)